File: programmers/숨어있는 숫자의 덧셈(2).py
def solution(my_string):
    answer = 0
    alphas = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    
    for i in alphas:
        if i in my_string:
            my_string = my_string.replace(i, ' ')
    
    for i in my_string.split():
        answer += int(i)
    
    return answer

File: programmers/숨어있는 숫자의 덧셈(2).cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string) {
    int answer = 0;
    vector<string> s_list;
    string tmp = "";
    for(int i = 0; i < my_string.size(); i++)
    {
        if(my_string[i] >= '0' && my_string[i] <= '9')
        {
            tmp += my_string[i];
        }
        else
        {
            if(tmp.compare("") != 0)
                s_list.push_back(tmp);
            tmp = "";
        }
    }
    if(tmp.compare("") != 0)
        s_list.push_back(tmp);
    for(int i = 0; i < s_list.size(); i++)
    {
        answer += stoi(s_list[i]);
    }
    return answer;
}